homeassistant/components/http/__init__.py

```diff
@@ -195,13 +199,28 @@ def __init__(self, hass, development, api_password, ssl_certificate,
         self.app[KEY_LOGIN_THRESHOLD] = login_threshold
         self.app[KEY_DEVELOPMENT] = development
 
+        try:
+            sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            sock.bind((server_host, server_port))
```

Comment:
Is this I/O and can this be blocking? This is getting initialized inside the event loop.